src/index.js

```diff
@@ -252,8 +252,15 @@ const urbitKeysFromSeed = (seed, password) => {
  */
 const fullWalletFromTicket = async config => {
   const { ticket, seedSize, ships, password, revisions, boot } = config;
-  const seed = await argon2u(ticket, seedSize).hash;
```

Comment:
This is so stupid. `await x.y` vs `(await x).y` has bitten me before. Maybe our style guide should forbid `await` without parens...

Good catch!

Reply:
@c-johnson you might be interested in what Jared found here.